docs/userguide/quickstart.txt

```diff
@@ -2,211 +2,189 @@
-managing the additional or optional dependencies, go to :ref:`entry_point`.
+to type ``python -m pip install``. To accomplish this, add the entry_points
+keyword in your ``setup.cfg``:
+
+.. code-block:: ini
+
+    [options]
+    entry_points =
+        [console_script]
+        main = mypkg:some_func
+
+When this project is installed, a ``main`` script will be installed and will
+invoke the ``some_func`` in the ``__init__.py`` file when called by the user.
+For detailed usage, including managing the additional or optional dependencies,
+go to :ref:`entry_point`.
+
 
 Dependency management
 =====================
-
 ``setuptools`` supports automatically installing dependencies when a package is
 installed. The simplest way to include requirement specifiers is to use the
-``install_requires`` argument to ``setup()``.  It takes a string or list of
-strings containing requirement specifiers::
+``install_requires`` argument to ``setup.cfg``.  It takes a string or list of
+strings containing requirement specifiers (A version specifier is one of the
+operators <, >, <=, >=, == or !=, followed by a version identifier):
+
+.. code-block:: ini
 
-    setup(
-        #...
-        install_requires = "docutils >= 0.3"
-    )
+    [options]
+    install_requires =
+        docutils >= 0.3
+        requests <= 0.4
 
-When your project is installed, either by using pip, ``setup.py install``,
-or ``setup.py develop``, all of the dependencies not already installed will
-be located (via PyPI), downloaded, built (if necessary), and installed.
+When your project is installed, all of the dependencies not already installed
+will be located (via PyPI), downloaded, built (if necessary), and installed.
+This, of course, is a simplified scenarios. ``setuptools`` also provide
+additional keywords such as ``setup_requires`` that allows you to install
+dependencies before running the script, and ``extras_requires`` that take
+care of those needed by automatically generated scripts. It also provides
+mechanisms to handle dependencies that are not in PyPI. For more advanced use,
+see :ref:`dependency_management`
 
-For more advanced use, see :ref:`dependency_management`
 
 Including Data Files
 ====================
-
 The distutils have traditionally allowed installation of "data files", which
 are placed in a platform-specific location. Setuptools offers three ways to
 specify data files to be included in your packages. For the simpliest use, you
-can simply use the ``include_package_data`` keyword e.g.::
+can simply use the ``include_package_data`` keyword:
+
+.. code-block:: ini
 
-    setup(
-            ...
-            include_package_data=True
-        )
+    [options]
+    include_package_data = True
 
 This tells setuptools to install any data files it finds in your packages.
 The data files must be specified via the distutils' ``MANIFEST.in`` file.
-
 For more details, see :ref:`datafiles`
 
-Development mode
```

Comment:
It might be nice to keep a section about this, except replace it with a brief direction to `pip install --editable .`

Reply:
I read that pep 517 is incompatible with editable install. should i point that out if I were to include this section?
<img width="1071" alt="Screen Shot 2020-05-25 at 5 31 34 PM" src="https://user-images.githubusercontent.com/53921639/82844199-a8fe7280-9ead-11ea-8c9d-3033ced979f8.png">

Reply:
Oh! I knew that was the case. Yeah, a setup.py is required in that case. Perhaps that should be the recommendation then (to include a nominal setup.py) to support development installs.

Reply:
i.e. https://github.com/jaraco/skeleton/blob/169dad91b8316a7138bec2f7b51db80cab0aadc4/setup.py#L1

Reply:
got it. thanks:)